pandas/core/ops.py

```diff
@@ -582,6 +577,155 @@ def _get_op_name(op, special):
+
+Notes
+--------
+Mismatched indices will be unioned together.
+
+Examples
+--------
+>>> df = pd.DataFrame({{'company': ['A', 'B', 'C'],
+...                    'cost': [250, 150, 100],
+...                    'revenue': [100, 250, 300]}})
+>>> df
+  company  cost  revenue
+0       A   250      100
+1       B   150      250
+2       C   100      300
+
+Compare to a scalar and operator version which return equivalent
+results.
+
+>>> df == 100
+   company   cost  revenue
+0    False  False     True
+1    False  False    False
+2    False   True    False
+>>> df.eq(100)
+   company   cost  revenue
+0    False  False     True
+1    False  False    False
+2    False   True    False
+
+Compare to a list and Series by axis and operator version.
+
+>>> df != [100, 250, 300]
+   company  cost  revenue
+0     True  True    False
+1     True  True    False
+2     True  True    False
+>>> df.ne([100, 250, 300], axis='index')
+   company  cost  revenue
+0     True  True    False
+1     True  True    False
+2     True  True    False
+>>> df != pd.Series([100, 250, 300])
+   company  cost  revenue     0     1     2
+0     True  True     True  True  True  True
+1     True  True     True  True  True  True
+2     True  True     True  True  True  True
+>>> df.ne(pd.Series([100, 250, 300]), axis='columns')
+   company  cost  revenue     0     1     2
+0     True  True     True  True  True  True
+1     True  True     True  True  True  True
+2     True  True     True  True  True  True
+
+Compare to a DataFrame of different shape by axis and operator version.
+
+>>> other = pd.DataFrame({{'company': ['A', 'B', 'C', 'D'],
+...                       'revenue': [300, 250, 100, 150]}})
+>>> other
+  company  revenue
+0       A      300
+1       B      250
+2       C      100
+3       D      150
+>>> df.reindex(['company', 'revenue'], axis='columns') > other.iloc[:-1]
+   company  revenue
+0    False    False
+1    False    False
+2    False     True
+>>> df.gt(other, axis=0)
+   company   cost  revenue
+0    False  False    False
+1    False  False    False
+2    False  False     True
+3    False  False    False
+>>> df.gt(other, axis=1)
+   company   cost  revenue
+0    False  False    False
+1    False  False    False
+2    False  False     True
+3    False  False    False
+
+Compare to a MultiIndex by level and operator version.
+
+>>> df_multindex = pd.DataFrame({{'cost': [250, 150, 100, 150, 300, 220],
+...                              'revenue': [100, 250, 300, 200, 175, 225]}},
+...                              index = [['Q1', 'Q1', 'Q1', 'Q2', 'Q2', 'Q2'],
+...                                       ['A', 'B', 'C', 'A', 'B' ,'C']])
+>>> df_multindex
+      cost  revenue
+Q1 A   250      100
+   B   150      250
+   C   100      300
+Q2 A   150      200
+   B   300      175
+   C   220      225
+
+>>> df.set_index('company') <= df_multindex.loc['Q1']
+         cost  revenue
+company
+A        True     True
+B        True     True
+C        True     True
```

Comment:
I think here we're comparing two `DataFrame` objects. This is more useful to show how `.loc` works with a multiindex than how operators behave. I'd get rid of it.